### PR TITLE
Feature/update drupal and php

### PR DESCRIPTION
**Pull request by martinyde**
https://jira.itkdev.dk/browse/LOOP-1272

- Upgrade drupal core (9.5.10)
- Upgrade contrib module
- Change code analysis tool (drupal-check -> phpstan for more configuration options)
- Update custom modules with phpcs and phpstan tools.
- Update configuration to match drupal upgrade.
- Upgrade docker setup to use php 8.1

Still missing:
- [ ] Use of php image with wkhtmltopdf tool included.
- [ ] Full docker setup for servers.

### CODE REVIEW

File: .DS_Store
Comment (rimi-itk) [inline .DS_Store:1]: `.DS_Store` files should never be added to git. I can recommend adding them to your global gitignore file (cf. https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/).

File: composer.json [hunk 1]
```diff
@@ -108,7 +112,7 @@
         ],
         "coding-standards-check": [
             "@coding-standards-check/phpcs",
-            "@coding-standards-check/drupal-check",
+            "@code-analysis",
```
Comment (rimi-itk) [inline composer.json:115]: We should remove “code analysis” from “coding standards check” and call “code analysis” when needed.

File: config/sync/entity_print.print_engine.dompdf.yml
Comment (rimi-itk) [inline config/sync/entity_print.print_engine.dompdf.yml:1]: This file hints at us being able to add Dompdf to the mix and get rid of wkhtmltopdf.

File: scripts/git/hooks/pre-commit
Comment (rimi-itk) [inline scripts/git/hooks/pre-commit:1]: I think we should remove this pre-commit hook. In my experience it causes more problems than it solves.

**Review by rimi-itk — CHANGES_REQUESTED** [review]
Looks good.

File: .DS_Store
Comment (martinyde) [reply .DS_Store:1]: Removed and added to gitignore.

File: composer.json (hunk 1 above)
Comment (martinyde) [reply composer.json]: Removed

File: scripts/git/hooks/pre-commit
Comment (martinyde) [reply scripts/git/hooks/pre-commit:1]: REmoved

**Review by rimi-itk — APPROVED** [review]

**Pull request merged**